src/suites/suite.ts

```diff
@@ -0,0 +1,33 @@
+import { OperationResult } from '../validation';
+import { TestOptions } from '../config/config.interface';
+import { SecurityValues } from 'swagger-client';
+import OASSchema from '../oas-parsing/schema';
+
+export interface SuiteConfig {
+  options: TestOptions;
+  schema: OASSchema;
+  securityValues?: SecurityValues;
+}
+
+// New test suites should be added under the '/suites/<suite name>' folder and defined in the SuiteFactory
```

Comment:
The naming is trying to follow terminology of other unit test conventions encountered in the past.  
- Test is just a single check against given criteria
- Test suite is a collection of related tests.  I've grouped the tests into the two test suites based on what they are attempting to test against. Example group req/resp checking vs linting.  In our case this is extra handy since different tooling drives the two test suites.
- Suite set being a combination of test suites.  Not all APIs should be getting the same test suites at the end of the day.
- Batch being a group of samples(API/ver.#) that a set of test are applied to.

The only thing that changed for the commands was they no longer run a the positive suite but instead they run a 'suite set'.

Reply:
If wanted, we could rename the commands to "suitesset" vs a "suitesset-batch".  I personally find it more confusing.